tests/framework/utils/s3_operations.go

```diff
@@ -24,12 +24,15 @@ import (
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/coreos/pkg/capnslog"
 )
 
 type S3Helper struct {
 	s3client *s3.S3
 }
 
+var s3logger = capnslog.NewPackageLogger("github.com/rook/rook", "s3utils")
```

Comment:
why have an s3-specific logger? if you need a specific error, you could include `s3` in the message.